PracticeWPF/PracticeWPF/ViewModel/MainWindowViewModel.cs

```diff
@@ -0,0 +1,106 @@
+﻿using PracticeWPF.Model;
+using PracticeWPF.ViewModel.Command;
+using System;
+using System.Collections.Generic;
+using System.Collections.ObjectModel;
+using System.ComponentModel;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using System.Windows.Input;
+using System.Windows.Media;
+
+namespace PracticeWPF.ViewModel
+{
+    public class MainWindowViewModel : INotifyPropertyChanged
+    {
+        private ObservableCollection<Product> products;
+
+        private Product selectedProduct;
+
+        public event PropertyChangedEventHandler PropertyChanged;
+
+        public ICommand DeleteCommand { get; set; }
+
+        public ICommand DefaultCommand { get; set; }
+
+        public Product SelectedProduct
+        {
+            get
+            {
+                return this.selectedProduct;
+            }
+            set
+            {
+                if (this.selectedProduct == value)
+                    return;
+
+                this.selectedProduct = value;
+                this.OnPropertyChanged(nameof(this.SelectedProduct));
+            }
+        }
+
+        public ObservableCollection<Product> Products
+        {
+            get
+            {
+                return this.products;
+            }
+            set
+            {
+                if (this.products == value)
+                    return;
+
+                this.products = value;
+                this.OnPropertyChanged(nameof(this.Products));
+            }
+        }
+
+        public MainWindowViewModel()
+        {
+            this.Products = new ObservableCollection<Product>()
+            {
+                new Product("TV", "LG", "CX25167","Korea", "FullHD", "Black", 15000.00m, @"C:\Users\Admin\source\repos\PracticeWPF\PracticeWPF\bin\Debug\Pictures\TVImage.jpg"),
```

Comment:
Object creation should have been placed in a separate class with an appropriate responsibilities.

Reply:
-1 to Concerns separation.